plugins/fmk.py

```diff
@@ -0,0 +1,21 @@
+import codecs
+import os
+import random
+import re
+
+from cloudbot import hook
+
+@hook.on_start()
+def load_fmk(bot):
+    """
+    :type bot: cloudbot.bot.Cloudbot
+    """
+    global fmklist
+
+    with codecs.open(os.path.join(bot.data_dir, "fmk.txt"), encoding="utf-8") as f:
+        fmklist = [line.strip() for line in f.readlines() if not line.startswith("//")]
+
+@hook.command("fmk", autohelp=False)
+def fmk(text, action):
+    """Fuck, Marry, Kill"""
+    message((text.strip() if is_valid(text.strip()) else "") + random.choice(fmklist) + ", " + random.choice(out) + ", " + random.choice(out))
```

Comment:
the is_valid() function is not defined in this file. Also the random.choice(out) will cause an error.